src/pages/Wallet/SendPage.tsx

```diff
@@ -97,32 +101,83 @@ const SendPage = () => {
         )
 
         const { txId, unsignedTx } = txCreateResp.data
+        const txSendResp = await signAndSendTransaction(txId, unsignedTx, address, wallet.privateKey)
+
+        if (txSendResp) {
+          addPendingTx({
+            txId: txSendResp.data.txId,
+            toAddress: address,
+            timestamp: new Date().getTime(),
+            amount: fullAmount,
+            type: 'transfer'
+          })
+        }
 
-        const signature = client.clique.transactionSign(txId, wallet.privateKey)
+        setSnackbarMessage({ text: 'Transaction sent!', type: 'success' })
+        history.push('/wallet')
+      } catch (e) {
+        // TODO: When API error codes are available, replace this substring check with a proper error code check
+        if (isHTTPError(e) && e.error?.detail && e.error.detail.includes('consolidating')) {
+          setIsConsolidateUTXOsModalVisible(true)
```

Comment:
When UTXO modal is visible, will it be sticky to top? I mean we can not bring the main window of wallet to the front

Reply:
Yes, it is sticky to the front. To close it (in order to access the rest of the wallet), you either have to click "Consilidate" or the "X" button.